var/spack/repos/builtin/packages/dla-future/package.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Comment:
```suggestion
# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
```

Reply:
Sorry, fixed in https://github.com/spack/spack/pull/38224/commits/f700b79c205361607915ffbd7e2a911117635134